analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BlazorQueryParameterRoutableComponent_Partial.razor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+
+namespace EmptyProject
```

Comment:
Optional nitpick: we can use file-scoped namespaces (in other test files as well)

Reply:
I won't update to file-scoped namespace but I added in all files.